store/sqlstore/scheme_store.go

```diff
@@ -301,24 +314,35 @@ func (s *SqlSchemeStore) Delete(schemeId string) (*model.Scheme, error) {
 	var inQueryList []string
 	queryArgs := make(map[string]interface{})
 	for i, roleId := range roleNames {
-		inQueryList = append(inQueryList, fmt.Sprintf(":RoleName%v", i))
-		queryArgs[fmt.Sprintf("RoleName%v", i)] = roleId
+		inQueryList = append(inQueryList, fmt.Sprintf(":rolename%v", i))
+		queryArgs[fmt.Sprintf("rolename%v", i)] = roleId
 	}
 	inQuery := strings.Join(inQueryList, ", ")
 
 	time := model.GetMillis()
-	queryArgs["UpdateAt"] = time
-	queryArgs["DeleteAt"] = time
+	queryArgs["updateat"] = time
+	queryArgs["deleteat"] = time
 
-	if _, err := s.GetMaster().Exec("UPDATE Roles SET UpdateAt = :UpdateAt, DeleteAt = :DeleteAt WHERE Name IN ("+inQuery+")", queryArgs); err != nil {
+	if _, err := s.GetMasterX().NamedExec(`UPDATE Roles SET UpdateAt = :updateat, DeleteAt = :deleteat WHERE Name IN (`+inQuery+`)`, queryArgs); err != nil {
```

Comment:
I have kept all the names in lowercase due to sqlx wrapper for map string interface. 
Reference :- https://github.com/jmoiron/sqlx/issues/465

Reply:
This query is actually a great candidate to rewrite in squirrel if you are interested :). Otherwise, this is okay as well.